Sync

mazipan opened the pull request.

mul14 commented:
Hmm... tidak bisa langsung di-merge dari GitHub nih. :|

mul14 commented:
Ini gak bisa di-merge. Via command line dapat message ini 

```
fatal: refusing to merge unrelated histories
```

Di repo anda tidak ada commit ini https://github.com/vuejs-id/docs/commit/8bdcd864fd87c59d684a5d398fd0e4733175b12b

agasigp commented:
Kalau repo fork nya dihapus, trus aku fork ulang lagi gimana? Terus aku commit ulang ke repo yg baru, baru aku kirim PR.

mul14 commented:
Bisa aja. Lain kali kalau mau send pull request, bikin di branch baru. Misalnya mau nambah fitur hapus.

```
git checkout --branch add-delete-feature
# Edit some files
git push origin add-delete-feature
```

Lalu yang dijadikan PR branch `add-delete-feature`. Jadi kalau PR `master` branch di-reject, tidak perlu fork ulang. Cukup hapus branch `add-delete-feature`.

Detailnya silahkan pelajari https://guides.github.com/introduction/flow/

agasigp commented:
Oke. Sekalian aku tambahin cara kontribusi sama halaman/fitur mana aja yg sudah/belum selesai diterjemahkan.

pieer committed:
Revert "Vue Mastery Free Weekend announcement (#1997)"

This reverts commit 23224fd2d6720a64f758204830ca5ee36c8e3edc.

yyx990803 committed:
add a note about dynamic arguments in in-DOM templates

posva committed:
Add Function type to props (#2026)

* Add Function type to props

* add Promise contstructor prop type

NataliaTepluhina committed:
Update core focus for myself (#2025)

* Add docs repo as core focus 

- and apparently fixed James McGlasson photo xD

NataliaTepluhina committed:
Added options for object-based props definition in API (#2028)

* feat: add API entry on props options

- fixed a link to props section

* fix: moved a link and added read-more types link

* Update src/v2/api/index.md

Co-Authored-By: NataliaTepluhina <NataliaTepluhina@users.noreply.github.com>

* Update src/v2/api/index.md

Co-Authored-By: NataliaTepluhina <NataliaTepluhina@users.noreply.github.com>

* Update src/v2/api/index.md

Co-Authored-By: NataliaTepluhina <NataliaTepluhina@users.noreply.github.com>

zdsfwy committed:
to reduce misleading, add a note for [v-for with v-if] section (#2029)

* to reduce misleading, add a note for [v-for with v-if] section

* move note to the top of the target section

twickstrom committed:
Globally register components example (#1959)

* Automatic Global Registration of Base Components example did not work if you wanted to include subfolders. This solution has been tested and confirmed. https://stackoverflow.com/questions/53676837/registering-components-globally-in-vuejs-in-subfolders/53678945#53678945

* Updated regex in global compnent registration code example to allow setting subfolder to true

NataliaTepluhina committed:
Merge pull request #2031 from NataliaTepluhina/master

Revert "Vue Mastery Free Weekend announcement (#1997)"

cristijora committed:
Add themes example (#2032)

Add themes example page with multiple free & paid Vue.js themes.
Add option to display that certain links are new/updated

Justineo committed:
Fix example typo

NataliaTepluhina committed:
Merge pull request #2033 from vuejs/fix-typo

Fix example typo

vue-bot committed:
chore: update sponsors (#2036)

NataliaTepluhina committed:
fix: fixed smooth scrolling (#2038)
